### PR TITLE
Enable MSP configuration on MSP peripherals

### PR DESCRIPTION
**Pull request by haslinghuis**
Enable MSP configuration on peripherals with MSP communication automatically after save and reboot.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
